modules/boss/src/main/ts/ephox/boss/api/DomUniverse.ts

```diff
@@ -48,6 +48,14 @@ export default (): Universe<SugarElement, Document> => {
     ], tag);
   };
 
+  const getLanguage = (element: SugarElement<Node>): Optional<string> => {
```

Comment:
I'd use a ternary here but it's just code style :)

`return SugarNode.isElement(element) ? Attribute.getOpt(element, 'lang') : Optional.none();`